data/json/mutations/mutations.json

```
@@ -1180,9 +1180,9 @@
   {
     "type": "mutation",
     "id": "PSYCHOPATH",
-    "name": { "str": "Psychopath" },
+    "name": { "str": "Uncaring" },
     "points": 2,
-    "description": "You don't experience guilt like others do.  Even when you know your actions are wrong, you just don't care.",
+    "description": "Whether due to brain chemistry, service-related trauma, or Olympian mental gymnastics, you no longer feel any remorse or guilt for harming your fellow man.",
```

I understand the desire to try to ground the effect somehow, but it's really not representative of anything. So we shouldn't be suggesting it is.
```suggestion
    "description": "You don't feel any remorse or guilt for harming your fellow man.",
```